src/dnsimple-test/Services/CertificatesTest.cs

```diff
@@ -98,24 +98,42 @@ public void CertificatesResponse()
+                "1qaWPtJBbkBuzJIn8gmSG+6xGmywB6GKvP2IVkPQhPBpfc8wsTd26rbSBHnRIQal\n" +
+                "tk+W4aQZyIeXFARY+cRvpjeAtmpX0vwZkDMoEyhFomBfGxVfx6tSqdGlR88/x0By\n" +
+                "y5u7+xwkY+4jMt+wZi+wpXsScumB6DAC1PTYRvNFQy7Gcjqrc3EdzPsn3c9kLCNO\n" +
+                "3GCPJoWmT5Rtyd7FxjJiSIf7BDOi12BnblpSLwGvtu6Wrl+u9LJLj8zeCACwUiQG\n" +
+                "uvnP2lAl2YacNAgpql6C2eEnFjIub7Ul1QMUImQSDVy5dMd/UGQrOb0CAwEAAaAA\n" +
+                "MA0GCSqGSIb3DQEBCwUAA4IBAQA8oVxOrZCGeSFmKpNV4oilzPOepTVSWxXa19T7\n" +
+                "zD/azh6j6RBLZPpG4TFbpvjecum+1V7Y8ypIcwhRtlh5/zSbfJkjJsdCdZU9XZat\n" +
+                "T5YkOaxuCUCDajpRiyyKhHvrloTPKPXe5ygCq/Q23xm//VrXKArLSWVB9qWS6gDV\n" +
+                "k0y3/mIlTQ3mTgfYQySc3MPXvIgUoqmB8Ajfq1n3hSLgb1/OoKNfeVEWsON116cq\n" +
+                "bXvl63+XzPubj6KWZXZH/jhrs53fuLq3xyeeuOaPrn+2VceBVt4DCC9n0JS5wepl\n" +
+                "HDoVxtWTTNeJdP5xFB5V1KI+D4FEFBUGnQABEvajpU3vljh3\n" +
+                "-----END CERTIFICATE REQUEST-----\n";
 
             Assert.Multiple(() =>
             {
                 Assert.AreEqual(2, certificate.Count);
-                Assert.AreEqual(1, certificate.First().Id);
-                Assert.AreEqual(10, certificate.First().DomainId);
-                Assert.AreEqual(3, certificate.First().ContactId);
-                Assert.AreEqual("www", certificate.First().Name);
-                Assert.AreEqual("www.weppos.net", certificate.First().CommonName);
+                Assert.AreEqual(101973, certificate.First().Id);
+                Assert.AreEqual(14279, certificate.First().DomainId);
+                Assert.AreEqual(11435, certificate.First().ContactId);
+                Assert.AreEqual("www2", certificate.First().Name);
+                Assert.AreEqual("www2.dnsimple.us", certificate.First().CommonName);
                 Assert.AreEqual(1, certificate.First().Years);
-                Assert.AreEqual(CertificateContent, certificate.First().Csr);
+                Assert.AreEqual(expectedCSR, certificate.First().Csr);
                 Assert.AreEqual("issued", certificate.First().State);
                 Assert.IsFalse(certificate.First().AutoRenew);
                 Assert.IsEmpty(certificate.First().AlternateNames);
                 Assert.AreEqual("letsencrypt", certificate.First().AuthorityIdentifier);
-                Assert.AreEqual(CreatedAt, certificate.First().CreatedAt);
-                Assert.AreEqual(UpdatedAt, certificate.First().UpdatedAt);
-                Assert.AreEqual(ExpiresOn, certificate.First().ExpiresOn);
+                Assert.AreEqual(Convert.ToDateTime("2020-06-18T20:15:09Z"), certificate.First().CreatedAt);
+                Assert.AreEqual(Convert.ToDateTime("2020-06-18T20:30:08Z"), certificate.First().UpdatedAt);
+                Assert.AreEqual(Convert.ToDateTime("2020-09-16T19:30:07Z"), certificate.First().ExpiresAt);
+                Assert.AreEqual("2020-09-16", certificate.First().ExpiresOn);
```

Comment:
Shouldn't we be using the values declared previously up here?

```suggestion
                Assert.AreEqual(Convert.ToDateTime("2020-06-18T20:15:09Z"), certificate.First().CreatedAt);
                Assert.AreEqual(Convert.ToDateTime("2020-06-18T20:30:08Z"), certificate.First().UpdatedAt);
                Assert.AreEqual(Convert.ToDateTime("2020-09-16T19:30:07Z"), certificate.First().ExpiresAt);
                Assert.AreEqual(Convert.ToDateTime("2020-09-16"), certificate.ExpiresOn);
```